contracts/VirtualSwap/README.md

```diff
@@ -0,0 +1,5 @@
+# Virtual swap
+
+Set of contracts used for swapping using Synthetix protocol as a mediatory swap.
+
+As of Jan 19, 2023, this is being phased out in lieu of Synthetix's atomic swaps.
```

Comment:
Do we want to explicitly state that this is out of scope for a bounty, or is that sufficiently implied by you saying that its being phased out?

Reply:
Yeah I can add that here